### PR TITLE
Lint API usage

### PR DESCRIPTION
**Pull request by fregante**
This PR

- Simplifies the API import
- Adds an uncached method to avoid the "cache buster"
- Drops an old unused header

### CODE REVIEW

File: source/features/previous-version.tsx [hunk 1]
```diff
@@ -5,22 +5,27 @@ import {VersionsIcon} from '@primer/octicons-react';
...
 import observe from '../helpers/selector-observer.js';
-import * as api from '../github-helpers/api.js';
+import api from '../github-helpers/api.js';
 import GitHubURL from '../github-helpers/github-url.js';
 
 async function getPreviousCommitForFile(pathname: string): Promise<string> {
 	const {user, repository, branch, filePath} = new GitHubURL(pathname);
 	const {resource} = await api.v4(`
-		resource(url: "/${user}/${repository}/commit/${branch}") {
+		resource(url: $resouce) {
```
Comment (fregante) [inline source/features/previous-version.tsx:14]: Typo

File: source/features/repo-age.tsx [hunk 2]
```diff
@@ -56,7 +56,11 @@ async function getRepoAge(commitSha: string, commitsCount: number): Promise<[com
 				}
 			}
 		}
-	`);
+	`, {
+		variables: {
+			cursor: `${commitSha} ${commitsCount - Math.min(6, commitsCount)}`,
```
Comment (fregante) [inline source/features/repo-age.tsx:61]:
```suggestion
			cursor: commitSha + ' ' + commitsCount - Math.min(6, commitsCount),
```

File: source/github-helpers/api.ts [hunk 3]
```diff
@@ -90,10 +90,13 @@ type GHRestApiOptions = {
 	body?: JsonObject;
 	headers?: HeadersInit;
 	json?: boolean;
+	cache?: boolean;
 };
 
 type GHGraphQLApiOptions = {
 	allowErrors?: boolean;
+	cache?: boolean;
```
Comment (fregante) [inline source/github-helpers/api.ts:98]:
```suggestion
	cache?: false;
```

File: source/github-helpers/api.ts [hunk 4]
```diff
@@ -90,10 +90,13 @@ type GHRestApiOptions = {
 	body?: JsonObject;
 	headers?: HeadersInit;
 	json?: boolean;
+	cache?: boolean;
```
Comment (fregante) [inline source/github-helpers/api.ts:93]: Not yet implemented

File: source/github-helpers/api.ts [hunk 5]
```diff
@@ -164,7 +164,7 @@ export const v3paginated = async function * (
 	}
 };
 
-export const v4 = mem(async (
+export const v4uncached = async (
```
Comment (fregante) [inline source/github-helpers/api.ts:167]: If anyone is reviewing this PR post-merge… naming suggestions still welcome 😅

Comment (yakov116) [reply source/github-helpers/api.ts:167]: Happen to think its good

**Pull request merged**